submissions/team_api.py

```diff
@@ -192,6 +191,40 @@ def _log_team_submission(team_submission_data):
     )
 
 
+def get_submissions_from_other_teams(
```

Comment:
The name doesn't "jive" with what the method actually does. Trying to think of a better one.

Reply:
I agree, this name sounds like it's getting the Submission  objects

get_teammates_with_submissions_from_other_teams() ?